var/spack/repos/builtin/packages/libyogrt/package.py

```diff
@@ -51,10 +53,37 @@ def configure_args(self):
         args = []
 
         sched = self.spec.variants['scheduler'].value
+        print("SCHEDULER=%s\n" % sched)
```

Comment:
I'm guessing this is a debug message, right?  If so, we can delete this print line before the merge.